### PR TITLE
HIP 56 - Improved State Channel Disputes

### PR DESCRIPTION
**Pull request by jthiller**
Sets up HIP 56 for a 5-day vote; terminating on block 1256800 (approx Sunday morning PST)
image

---
note to @danielcolinjames: 0c9a4da61df7b0bb6c36eb730fa6824fa862be37

### CODE REVIEW

**Review by danielcolinjames — APPROVED** [review]
looks good!

File: next.config.js [hunk 1]
```diff
@@ -263,6 +263,49 @@ module.exports = {
...
+          {
+            nickname: "@abhay",
+            link: "https://github.com/abhay",
+          },
+          {
+            nickname: "et al."
+          }
+        ],
+        description:
+          "HIP 56 describes an update to the dispute mechanism for State Channels that would improve blockchain scaling. These changes allow Validators to better reconcile Data Transfer activity and rewards for Hotspots. The change would also address recent issues around large block sizes. This vote is for the new dispute mechanism to be enabled by a chain variable.\n\nApproving this HIP implies that the community has reviewed the implementation and the relevant Chain Variables. We will consider this proposal as approved if it passes with 2/3 of the vote power by the deadline.\n\nJoin the conversation on Discord and see the #hip‑56‑improved‑state‑channel‑disputes channel for more details.",
```
Comment (abhay) [inline next.config.js:297]:
```suggestion
          "HIP 56 describes an update to the dispute mechanism for State Channels that will enable better accounting of State Channel activity and more accurate rewards for Data Transfer while improving blockchain scaling. It specifically protects against an edge case that could cause blockchain halts due to the dispute mechanism. This vote is for the new dispute mechanism to be enabled by a chain variable.\n\nApproving this HIP implies that the community has reviewed the implementation and the relevant Chain Variables. We will consider this proposal as approved if it passes with 2/3 of the vote power by the deadline.\n\nJoin the conversation on Discord and see the #hip‑56‑improved‑state‑channel‑disputes channel for more details.",
```

Comment (abhay) [reply next.config.js:297]: hopefully more plain language

Comment (jthiller) [reply next.config.js:297]: Love it. Thank you!

**Pull request merged**